docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -1042,6 +1068,18 @@ Returns an action object used in signalling that the caret has entered formatted
 
 Returns an action object used in signalling that the user caret has exited formatted text.
 
+### selectionChange
```

Comment:
To maximize consistency with other action names in the pattern of "verbNoun" / "actSubject" (`insertBlock`, `updateSettings`, `clearSelectedBlock`), in retrospect I think this should have been named as `changeSelection`. Has this landed in a stable WordPress release?